django/api/models.py

```diff
@@ -21,6 +21,7 @@ class UserProfile(models.Model):
     GENDER_CHOICES = (
         ('M', 'Male'),
         ('F', 'Female'),
+        ('U', 'Undefined'),
```

Comment:
This comment is not blocking, But, what's the reason of this change?

Reply:
Check https://www.formisimo.com/blog/improve-user-experience-with-gender-neutral-options-in-webforms/

Reply:
This option is for people who dont want to share their gender with everyone in the internet. Im not sure if we need it at all though

Reply:
Third option could be "gender diverse"?

Reply:
https://www.quora.com/How-should-I-ask-about-gender-on-a-survey-in-a-non-binary-way

Reply:
Could also provide a text input form...